### DISCUSSION TITLE
Just installed NextJs and I got 6 high severity vulnerabilities,

### DISCUSSION THREAD

**Discussion by Tarbez** [vercel/next.js#42332] [Help]
I have installed Nextjs but I'm facing 6 high severity vulnerabilities and I pretty much looked everywhere on how to fix but I can't seem to overcome this issue. 
I'm using
  npm: '8.19.2',
  node: '18.12.0',

![Screenshot 2022-11-02 at 7 07 24](https://user-images.githubusercontent.com/33286159/199402901-0cc09338-780b-4bbd-aab6-133da0dda54f.png)

What should I do from here?

**Comment by icyJoseph** [comment]
`react-scripts` is from an other React application creation tool, called create-react-app, which is not the same as create-next-app

Can you share a repository of how you installed "next.js"? I don't think we are talking about the same thing:

```
delete-me on  main via  v16.14.0 at  from 
➜ yarn audit
yarn audit v1.22.19
0 vulnerabilities found - Packages audited: 37
✨  Done in 0.30s.
```

Just made a project Nov 2nd 7 am at my time zone. 0 vulnerabilities.

It'd still be careful with "vulnerabilities", https://overreacted.io/npm-audit-broken-by-design/ you got to understand how the "vulnerable" package is ultimately put to use, and if its ever really an open vector attack. Depending on who you ask, all packages are an attack vector because it is code you run, which you don't control.

**Reply by Tarbez** [reply]
After a day of research it seems the issue is with the NPM itself. Now, I neglected the NPM. Now, I use yarn and there are no issue anymore.
